### PR TITLE
feat/migrate to react

### PR DESCRIPTION
**Pull request by diasdavid**

### CODE REVIEW

File: columns-content.jade [hunk 1]
```diff
@@ -0,0 +1,131 @@
+section(id="aboutme")
```
Comment (sericaia) [inline columns-content.jade]: jade is now pug (pugjs.org)

File: src/components/app/app.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+section h1 {
+  color: #0474ba;
+  font-weight: 400;
+}
+
+section h2 {
+  color: #0474ba;
+  font-weight: 300;
+  font-size: 30px;
+
```
Comment (sericaia) [inline src/components/app/app.css]: remove extra \n

File: src/components/main-column/main-column.js [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+    return (
+      <div>
+        <section id='about-me'>
+          <a href='#about-me'><h1>about me</h1></a>
+          <p>I consider myself to be one from many™ lucky bastards™ that found their opportunity to work with the most hard working and passionate people in humbling and encouraging endeavour such as <a href='http://lxjs.org'>LXJS</a>, the Lisbon Javascript Conference and <a href='http://startupscholarship.org'>Startup Scholarship</a> the first startup internship summer program in Portugal with a summer school and immersion program. I've a MSc in Engineering with major in Peer-to-Peer Networks from <a href='http://tecnico.ulisboa.pt/en/'>Technical University of Lisbon</a>.</p>
+        </section>
+
+        <section id='latest'>
+          <a href='//blog.daviddias.me'><h1>latest on my blog</h1></a>
+          <div className='well well-sm'>
```
Comment (sericaia) [inline src/components/main-column/main-column.js]: Lots of repeating code here.. I would change this a bit creating an array such as the following and interate over it ;)
```
const latest = [
{
  href: 'http://blog.daviddias.me/2015/03/22/enter-webrtc-explorer',
  text: 'webrtc-explorer - Resource Discovery for decentralized browser networks',
  date: 'Mar 22 2014'
},
 //....
]
```

File: src/components/main-column/main-column.js [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+          </div>
+          <div className='well well-sm'>
+            <a href='http://blog.daviddias.me/2014/02/01/get-your-feet-wet-with-docker'>Feb 01 2014 | Get your feet wet with docker</a>
+          </div>
+        </section>
+
+        <section id='endeavours'>
+          <a href='#endeavours'><h1>endeavours</h1></a>
+          <ul>
+            <li id='lxjs'>LXJS - Lisbon Javascript Conference</li>
```
Comment (sericaia) [inline src/components/main-column/main-column.js]: same as before ;p

File: src/components/side-column/side-column.js [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+          </p>
+        </section>
+
+        {/*
+        <section>
+          <a
+            className='twitter-timeline'
+            href='https://twitter.com/daviddias'
+            data-widget-id='546661201862594562'>Tweets by @daviddias</a>
+          <script dangerouslySetInnerHTML={{ __html:
```
Comment (sericaia) [inline src/components/side-column/side-column.js]: Can't you use this: https://www.npmjs.com/package/react-twitter-widgets ? 
I don't like the idea of having `dangerouslySetInnerHTML`

File: src/components/side-column/side-column.js [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+            `
+            !function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document,'script','twitter-wjs');
+            `}} />
+        </section>
+        */}
+
+        <section>
+          <iframe
```
Comment (sericaia) [inline src/components/side-column/side-column.js]: using `iframe` isn't generally a good idea.. isn't there any cool component doing this?

File: columns-content.jade (hunk 1 above)
Comment (sericaia) [inline columns-content.jade]: (do you still need this file?)

File: src/components/side-column/side-column.js (hunk 5 above)
Comment (diasdavid) [inline src/components/side-column/side-column.js]: oh, nice! :D

File: src/components/side-column/side-column.js (hunk 6 above)
Comment (diasdavid) [inline src/components/side-column/side-column.js]: Haven't found one yet, focused on 'migrating' first. Maybe I need to make one :)

File: package.json [hunk 7]
```diff
@@ -2,30 +2,31 @@
   "name": "daviddias.me",
   "version": "1.0.0",
   "private": false,
-  "description": "David Dias personal page",
-  "main": "./index",
+  "main": "src/index.js",
+  "homepage": "./",
```
Comment (diasdavid) [inline package.json]: @sericaia @fampinheiro This did the trick :)

//ref https://github.com/facebookincubator/create-react-app/issues/1094

Comment (gaearon) [inline package.json]: Why was this necessary? Are you deploying to different target URLs?
